src/winrpm_bindeps.jl

```diff
@@ -15,28 +15,29 @@ function package_available(p::RPM)
     global update_once
     !can_use(RPM) && return false
     pkgs = p.package
-    if isa(pkgs,AbstractString)
+    if isa(pkgs, AbstractString)
         pkgs = [pkgs]
     end
     if (update_once::Bool)
         info("Updating WinRPM package list")
-        update(); update_once = false;
+        update() 
+        update_once = false
     end
-    return all(pkg->(length(lookup(pkg).p) > 0),pkgs)
+    return all(pkg -> (length(lookup(pkg).p) > 0), pkgs)
 end
 
 available_version(p::RPM) = lookup(p.package).p[1][xpath"version/@ver"][1]
-libdir(p::RPM,dep) = joinpath(dirname(dirname(@__FILE__)),"deps","usr","$(Sys.ARCH)-w64-mingw32","sys-root","mingw","bin")
+libdir(p::RPM,dep) = joinpath(dirname(dirname(@__FILE__)), "deps", "usr", "$(Sys.ARCH)-w64-mingw32", "sys-root", "mingw", "bin")
 pkg_name(p::RPM) = p.package
 
-provider(::Type{RPM},packages::Vector{Compat.ASCIIString}; opts...) = RPM(packages)
-provides(::Type{RPM},packages::AbstractArray, dep::LibraryDependency; opts...) =
+provider(::Type{RPM}, packages::Vector{AbstractString}; opts...) = RPM(packages)
```

Comment:
sure, but if these are user facing then they need to handle types that will come from user input or code